debezium-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "timescaledb-sink",
+  "config": {
+    "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector",
+    "tasks.max": "1",
+    "topics": "gold-topic",
+    "connection.url": "jdbc:postgresql://timescaledb:5432/postgres?user=postgres&password=houssem",

```

Comment:
It's a good practice to not put user and password in an .env file and then use them here instead of writing them directly!